tensorboard/webapp/routes/dashboard_deeplink_provider_test.ts

```diff
@@ -391,6 +391,27 @@ describe('core deeplink provider', () => {
     ]);
   });
 
+  it('dedupes unknown query params with feature flags', () => {
+    store.overrideSelector(selectors.getOverriddenFeatureFlags, {
+      enabledExperimentalPlugins: ['foo', 'bar', 'baz'],
+      showFlags: '',
+    });
+    store.overrideSelector(selectors.getUnknownQueryParams, {
+      foo: 'bar',
```

Comment:
Are these `foo` and `bar` query params related to the value of the `enabledExperimentalPlugins` flag?

I'm not sure I follow what is being tested here. What is being deduped? The `showFlags` one? Are the other flags relevant for the test, then?

Reply:
I tried to make the test clearer and not reuse 'foo' and 'bar'. Let me know if it makes more sense now.

Reply:
The new tests are clearer in that regard, thanks!